src/main/java/com/example/demo/user/Role.java

```diff
@@ -0,0 +1,39 @@
+package com.example.demo.user;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "role")
+@NoArgsConstructor
+@EqualsAndHashCode(of = "role")
+public class Role implements GrantedAuthority {
+    public static final String ROLE_USER = "user";
```

Comment:
It's better to extract roles to the separate enum. See for reference: https://www.baeldung.com/jpa-persisting-enums-in-jpa